1-exercises/F-array-forEach/exercise.js

```diff
@@ -8,6 +8,25 @@
 */
 
 let arr = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15];
+function checkMultiple(item){
+  if(item%5===0 && item%3===0){
+    return 'FizzBuzz';
+  }
+  if(item%3===0){
+    return 'Fizz';
+  } 
+  if(item%5===0){
+    return 'Buzz';
+  }
+  else return item;
+}
+// console.log(arr.map(checkMultiple))
+arr.forEach((item)=>{
+  console.log(checkMultiple(item));
```

Comment:
Your code looks neat Delnia. I cannot find anything else to say except that you have done a good job of it.